source/_docs/guides/build-tools/03-create-pr.md

```diff
@@ -0,0 +1,88 @@
+   </div>
+ </div>
+
+
+1. From your GitHub project page, click on the `config` directory. Select the file named `system.site.yml` and click <span class="glyphicon glyphicon-pencil"></span> to open an editor:
+
+    ![system.site.yml Configuration](/source/docs/assets/images/pr-workflow/system-site-config.png)
+
+2. Change the slogan to something inspiring:
+
+    ![Edit slogan](/source/docs/assets/images/pr-workflow/edit-slogan.png)
+
+3. Scroll down and enter a message describing this change in the **Commit changes** area. Then, click on the radio button to create a **new branch** and give it a short name, like `slogan`, then click **Propose file change**:
+
+    ![Create slogan branch](/source/docs/assets/images/pr-workflow/create-slogan-branch.png)
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+      <div class="panel-heading panel-drop-heading">
+         <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#understand-branch-names"><h3 class="panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-lightbulb"></span> Branch Naming Conventions</h3></a>
+       </div>
+       <div id="understand-branch-names" class="collapse">
+         <div class="panel-inner" markdown="1">
+         Be sure to use unique branch names for each feature branch; the Multidev environment created on Pantheon is named after your branch. Since there is a limit to the number of characters that may be used in a Pantheon Multidev name, your environments may conflict if you always use the branch name that GitHub suggests. For details, see [Multidev](/docs/multidev/#what-are-the-naming-conventions-for-branches).
+        </div>
+       </div>
+     </div>
+
+
+4. Click **Create Pull Request**:
+
+    ![Slogan Pull Request](/source/docs/assets/images/pr-workflow/slogan-pull-request.png)
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+      <div class="panel-heading panel-drop-heading">
+         <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#understand-builds"><h3 class="panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-watch"></span> Builds</h3></a>
+       </div>
+       <div id="understand-builds" class="collapse">
+         <div class="panel-inner" markdown="1">
+         As soon as you commit changes to a feature branch, CircleCI builds a new Multidev environment on Pantheon to preview the change. Once the Multidev environment has been created, the build script will add a comment to the commit with links to the Multidev environment of the Pantheon Site Dashboard and a button to visit the Mutidev site URL (e.g., `pr-slogan-my-pantheon-project.pantheonsite.io`). The Pull Request page conveniently shows the messages from each commit on the branch:
```

Comment:
If we make the aforementioned change, then CircleCI will not build the new multidev environment until the pull request is created.